learnyounode/08-juggling-async.js

```diff
@@ -0,0 +1,35 @@
+var http = require('http');
+
+var urls = process.argv.slice(2);
+var results = [];
+
+// initialise results array
+for (i = 0; i < urls.length; i++) {
+  results[i] = null;
+}
```

Comment:
So, funnily enough, this step is actually not necessary in JavaScript -- you can just start inserting values into an empty array, and JS will fill the intermediary addresses with `undefined`.